### ISSUE TITLE
Typescript Errors

### ISSUE THREAD

**Issue by Industrial** [CSFrequency/react-firebase-hooks#28]
Hi.

I'm getting these:

```
node_modules/react-firebase-hooks/firestore/dist/firestore/useCollection.d.ts:2:29 - error TS2307: Cannot find module '../util'.

2 import { LoadingHook } from '../util';
                              ~~~~~~~~~

node_modules/react-firebase-hooks/firestore/dist/firestore/useCollectionOnce.d.ts:2:29 - error TS2307: Cannot find module '../util'.

2 import { LoadingHook } from '../util';
                              ~~~~~~~~~

node_modules/react-firebase-hooks/firestore/dist/firestore/useDocument.d.ts:2:29 - error TS2307: Cannot find module '../util'.

2 import { LoadingHook } from '../util';
                              ~~~~~~~~~

node_modules/react-firebase-hooks/firestore/dist/firestore/useDocumentOnce.d.ts:2:29 - error TS2307: Cannot find module '../util'.

2 import { LoadingHook } from '../util';
                              ~~~~~~~~~
```

**Comment by kaw2k** [comment]
Same issue here

**Comment by chrisbianca** [comment]
When are you getting these errors? 

I'll need some more details on your project setup to be able to debug the issue as this isn't affecting the majority of people.

**Comment by andipaetzold** [comment]
I have the same issue.
You can reproduce it by creating a new package and installing the library:
* `npm init`
* `npm i react-firebase-hooks`

As you can see in the directory tree, the utils are not bundled.
```
$ tree

├── node_modules
│   └── react-firebase-hooks
│       ├── LICENSE
│       ├── README.md
│       ├── auth
│       │   ├── README.md
│       │   ├── dist
│       │   │   ├── auth
│       │   │   │   ├── index.d.ts
│       │   │   │   └── useAuthState.d.ts
│       │   │   ├── index.cjs.js
│       │   │   ├── index.cjs.js.flow
│       │   │   ├── index.esm.js
│       │   │   └── index.esm.js.flow
│       │   └── package.json
│       ├── database
│       │   ├── README.md
│       │   ├── dist
│       │   │   ├── database
│       │   │   │   ├── helpers
│       │   │   │   │   ├── index.d.ts
│       │   │   │   │   └── useListReducer.d.ts
│       │   │   │   ├── index.d.ts
│       │   │   │   ├── useList.d.ts
│       │   │   │   └── useObject.d.ts
│       │   │   ├── index.cjs.js
│       │   │   ├── index.cjs.js.flow
│       │   │   ├── index.esm.js
│       │   │   └── index.esm.js.flow
│       │   └── package.json
│       ├── dist
│       │   ├── react-firebase-hooks-auth.js
│       │   ├── react-firebase-hooks-auth.js.map
│       │   ├── react-firebase-hooks-database.js
│       │   ├── react-firebase-hooks-database.js.map
│       │   ├── react-firebase-hooks-firestore.js
│       │   ├── react-firebase-hooks-firestore.js.map
│       │   ├── react-firebase-hooks-storage.js
│       │   └── react-firebase-hooks-storage.js.map
│       ├── firestore
│       │   ├── README.md
│       │   ├── dist
│       │   │   ├── firestore
│       │   │   │   ├── helpers
│       │   │   │   │   └── index.d.ts
│       │   │   │   ├── index.d.ts
│       │   │   │   ├── useCollection.d.ts
│       │   │   │   ├── useCollectionOnce.d.ts
│       │   │   │   ├── useDocument.d.ts
│       │   │   │   └── useDocumentOnce.d.ts
│       │   │   ├── index.cjs.js
│       │   │   ├── index.cjs.js.flow
│       │   │   ├── index.esm.js
│       │   │   └── index.esm.js.flow
│       │   └── package.json
│       ├── package.json
│       └── storage
│           ├── README.md
│           ├── dist
│           │   ├── index.cjs.js
│           │   ├── index.cjs.js.flow
│           │   ├── index.esm.js
│           │   ├── index.esm.js.flow
│           │   └── storage
│           │       ├── index.d.ts
│           │       └── useDownloadURL.d.ts
│           └── package.json
├── package-lock.json
└── package.json
```

**Comment by chrisbianca** [comment]
This has been fixed as part of v2.1.1.
Thanks to @miyucy for their PR to address this problem.
